src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -141,6 +149,10 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
                 context.EnvironmentVariables[DashboardConfigNames.DashboardOtlpAuthModeName.EnvVarName] = "Unsecured";
             }
 
+            // Change the dashboard formatter to use JSON so we can parse the logs and render them in the
+            // via the ILogger.
+            context.EnvironmentVariables["LOGGING__CONSOLE__FORMATTERNAME"] = "json";
```

Comment:
I like it. Better than my idea of copying the output directly to the host console.